runtime/src/accounts.rs

```diff
@@ -1741,72 +1747,6 @@ mod tests {
         }
     }
 
-    #[test]
```

Comment:
nit: perhaps instead of removal, can use `load_accounts_with_excluded_features(...)` to test call chain with/without feature_set::disable_builtin_loader_ownership_chains.